MdeModulePkg/Include/Library/FlatPageTableLib.h

```diff
@@ -0,0 +1,166 @@
+/** @file

```

Comment:
nit: Honestly not sure if we follow this but for the comment documentation tags @file, they should be followed by the filename.
e.g.
@file FlatPageTableLib.h

Or do we use a different system?

Reply:
In EDK2 headers, I only see @file. In some Project Mu headers, I see @file <filename>. I'll have to check if the EDK2 code format guidelines weigh in on this :)